modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -173,6 +175,45 @@ public void testGetRoleUserIds() throws Exception {
 		}
 	}
 
+	@Test
+	public void testUpdateOrganizationsWithoutPermission() throws Exception {
+		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+		RoleTestUtil.addResourcePermission(
+			role, Organization.class.getName(), ResourceConstants.SCOPE_COMPANY,
+			String.valueOf(TestPropsValues.getCompanyId()),
+			ActionKeys.ASSIGN_MEMBERS);
+		RoleTestUtil.addResourcePermission(
+			role, User.class.getName(), ResourceConstants.SCOPE_COMPANY,
+			String.valueOf(TestPropsValues.getCompanyId()), ActionKeys.UPDATE);
+
+		User user1 = UserTestUtil.addUser();
+		User user2 = UserTestUtil.addUser();
+
+		Organization organization = OrganizationTestUtil.addOrganization();
+
+		_userLocalService.addRoleUser(role.getRoleId(), user1.getUserId());
```

Comment:
please sort like this
```
User user1 = UserTestUtil.addUser();

_userLocalService.addRoleUser(role.getRoleId(), user1.getUserId());

User user2 = UserTestUtil.addUser();
Organization organization = OrganizationTestUtil.addOrganization();
```

Reply:
basically to reflect the usage
* user1 declaration
* user1 usage
* user2 and organization declaration in the order of usage inside 
```
_userService.updateOrganizations(
				user2.getUserId(),
				new long[] {organization.getOrganizationId()},
				ServiceContextTestUtil.getServiceContext());
```